### PR TITLE
Clean up PKGBUILD a bit

### PR DESCRIPTION
**Pull request by ISSOtm**
Hi! It's been a while. The recent comment on Aseprite breaking after the `{fmt}` 9.x update made me look into the PKGBUILD again, since it prints a warning during the build. This led to the following:

- Remove unused Skia config flags [and add a lengthy comment about *why* the flags are there]
- Disable two unnecessary Skia features [note: I did some light testing and it seemed fine, but I'm far from 100% coverage]
- Remove a duplicate LAF config flag
- Correct use of wrong variable in package()

Additionally, patches are currently applied with offsets, and one even with fuzz. Should we freshen those, or is is not worth the trouble?

### CODE REVIEW

File: aseprite/PKGBUILD [hunk 1]
```diff
@@ -82,21 +82,42 @@ prepare() {
...
+	#   skia_enable_pdf: Not used by Aseprite.
+	#   skia_enable_skottie: Not used by Aseprite.
+	#   skia_enable_sksl: laf seems to want to use it... but no references are made anywhere.
+	#   skia_enable_svg: Not used by Aseprite. It seems it has its own SVG exporter.
+	#   skia_use_lib*_{encode,decode}: Aseprite only loads PNG assets, so only libpng is required.
+	#   skia_use_expat: Only required for the Android font manager and SVGCanvas/SVGDevice.
+	#   skia_use_piex: Not used by Aseprite. Only used for reading RAW files.
+	#   skia_use_xps: Not used outside of Windows.
+	#   skia_use_zlib: Only used for PDF and RAW files.
+	#   skia_use_libgifcodec: Only used for GIFs, which Aseprite doesn't use.
```
Comment (ImperatorStorm) [inline aseprite/PKGBUILD:103]: Aseprite supports `gif`s quite well, unless that's changed in recent versions.

**Comment by ImperatorStorm** [comment]
Wrt patches, if its just offset its probably fine, though fuzz and outright breakage should be fixed.

**Comment by ImperatorStorm** [comment]
Also, I'm going to fix buildscript for PRs soon.

File: aseprite/PKGBUILD (hunk 1 above)
Comment (ISSOtm) [reply aseprite/PKGBUILD:103]: They do, but they load the GIFs into the editor; they never pass those to Skia. I checked that disabling `libpng` does produce an error on startup, citing an internal `sheet.png` file. Considering that there are only PNG files there, disabling support for other formats *in Skia* seems reasonable.

Comment (ImperatorStorm) [reply aseprite/PKGBUILD:103]: Gotcha.

**Comment by ImperatorStorm** [comment]
Not at computer so I can't check for myself, is it possible to version pin shared libraries in `PKGBUILD`?

**Comment by ISSOtm** [comment]
[This is done automatically at build time](https://wiki.archlinux.org/title/PKGBUILD#depends):

> If the dependency name appears to be a library, e.g. `depends=('libfoobar.so')`, makepkg will try to find a binary that depends on the library in the built package and append the soname version needed by the binary. Appending the version yourself disables automatic detection, e.g. `depends=('libfoobar.so=2')`.

**Comment by ImperatorStorm** [comment]
Can you rebase onto current `master`? Doesn't look like Github'll re-run PR CI on upstream's workflows.

**Comment by ImperatorStorm** [comment]
> https://github.com/ImperatorStorm/PKGBUILDs/pull/9#issuecomment-1236428755

Github's being a pain, and doesn't document how I would go about this as best I can tell.

Just gonna merge this and hope it works.

**Comment by ISSOtm** [comment]
Thanks!

**Pull request merged**